users/templates/users/create.html

```diff
@@ -1,11 +1,87 @@
+<style>
```

Comment:
ideally we should share the css for this and the login page in a static css file.
maybe add a head block to barbase.html?

Reply:
Done